recipes/spirv-headers/all/conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class SpirvheadersConan(ConanFile):
+    name = "spirv-headers"
+    homepage = "https://github.com/KhronosGroup/SPIRV-Headers"
+    description = "SPIRV-Headers"
```

Comment:
Please improve the description of the library

Reply:
Usually the GitHub repo's description is what I use for the conan description, but it didn't have one. Changed it to this:

Header files for the SPIRV instruction set.